relayer/crates/starknet-integration-tests/src/impls/build_bootstrap.rs

```diff
@@ -120,7 +125,7 @@ where
 
         let proof_signer = Secp256k1KeyPair::from_mnemonic(
             bip39::Mnemonic::from_entropy(
-                &relayer_wallet.signing_key.to_bytes_be(),
+                &relayer_wallet_1.signing_key.to_bytes_be(),
```

Comment:
what was `proof_signer` again ?

I think, this was part of the permissioned relaying.

Reply:
Oh good catch, it seems we don't use the proof signer anymore. I will remove it.

Reply:
Done here 0dde7f5f2b3bcda70e370504eb685c25047936d5